src/utils.js

```diff
@@ -0,0 +1,33 @@
+export function isValid(value) {
```

Comment:
try using more clear name, it's unclear what should be valid or not using this function